lib/hawk/model/association.rb

```diff
@@ -22,43 +22,68 @@ def initialize(attributes = {}, params = {})
       end
 
       private
+        # Called to kick off the preloading of association objects from the response hash
+        #
         def preload_associations(attributes, params, scope)
-          self.instance_exec(scope, attributes, &scope.preload_association)
+          self.instance_exec(scope, attributes, params, &scope.preload_association)
```

Comment:
Can you please update the default `preload_association` implementation below at line 144?